staging/src/k8s.io/api/core/v1/types.go

```diff
@@ -2026,7 +2026,7 @@ type EnvVar struct {
 	// Defaults to "".
 	// +optional
 	Value string `json:"value,omitempty" protobuf:"bytes,2,opt,name=value"`
-	// Source for the environment variable's value. Cannot be used if value is not empty.
+	// Source for the environment variable's value. Only used if value is empty.
```

Comment:
this sounds weird, could the original should have been `Cannot be used if value is empty.`?

Reply:
How about:
```suggestion
	// Source for this environment variable's value. You cannot set both value and valueFrom.
```

and make a similar change to the doc comment for `Value`